docs/data-sources/http.md

```diff
@@ -101,6 +101,8 @@ resource "random_uuid" "example" {
 
 ### Optional
 
+- `ca_certificate` - (String) PEM-encoded root certificate bundle for TLS authentication.
```

Comment:
Nitpick (but not that much): in the interest of consistency with other HC utility providers, how do you see using the name `ca_cert_pem` here instead?

TLS: https://registry.terraform.io/providers/hashicorp/tls/latest/docs/resources/locally_signed_cert#ca_cert_pem

There is also another reason: giving a hint about the type of string expected in there.